packages/vapor/resources/icons/svg/catalog.svg

```diff
@@ -0,0 +1 @@
+<svg  viewBox="0 0 52 52"><rect fill="#1E253F" width="52" height="52"></rect><path d="M25.6904,12.75 C25.3726147,12.75 25.0666061,12.8752541 24.8417301,13.1001301 L12.8784506,25.0508094 C12.4589656,25.4702946 12.2500133,26.0240608 12.249825,26.5770935 C12.2496369,27.1297894 12.4579744,27.6828337 12.8770012,28.1013412 L23.342401,38.5695409 C23.7615179,38.9896971 24.3153888,39.1982 24.8686,39.1982 C25.4221494,39.1982 25.9751197,38.9892201 26.3948699,38.5694699 L38.3453631,26.6231767 C38.5711506,26.3973893 38.6982,26.0922753 38.6982,25.7732 L38.581006,13.4664364 C38.2237501,13.3160481 37.622175,13.1948416 36.624953,13.0864634 C34.6813697,12.8752344 31.3594967,12.75 25.6904,12.75 Z" stroke="#FFFFFF" stroke-width="1.5"></path><polyline stroke="#FFFFFF" stroke-width="1.5" points="20.6176803 26 24.8658677 30.1299744 33.092175 21.8700256"></polyline></svg>
```

Comment:
The other source icons are prefixed with `source` should we do that too?
https://vapor.cloud.coveo.com/feat-com-365-add-catalog-icon/index.html#/styles/icons/list